Source/ViewerController.swift

```diff
@@ -94,12 +94,12 @@ public class ViewerController: UIViewController {
     /**
```

Comment:
Changes on ViewerController are mainly to support the functionality on PhotosController.